app/src/main/java/com/sacramento/movies/ui/movies/MoviesFragment.kt

```diff
@@ -122,4 +126,14 @@ class MoviesFragment : Fragment() {
 
     }
 
+    private fun refreshList() {
+        binding.pullRefreshLayout.setOnRefreshListener {
+            job?.cancel()
+            job = null
```

Comment:
Job operation could be moved into setupMoviesList
The same comment for SearchMovieByNameFragment

Reply:
Fixed